tools/src/protoc-gen-go/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/envoyproxy/gateway/tools/src/protoc-gen-go
```

Comment:
hey @AliceProxy any idea why this wasn't added in the orig PR
https://github.com/envoyproxy/gateway/pull/1209/

Reply:
cc @haq204

Reply:
Huh... I remember seeing that when I was messing around with stuff... maybe I forgot to push it or something when I was trying to figure out a cleaner way to handle the buf files?

Reply:
cool thanks for confirming !